react/features/invite/components/dial-in-info-page/NumbersTable.web.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { translate } from '../../../base/i18n';
+
+/**
+ * Displays a table with phone numbers to dial in to a conference.
+ *
+ * @extends Component
+ */
+class NumbersTable extends Component {
+    /**
+     * {@code NumbersTable} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The phone numbers to display. Can be an array of numbers
+         * or an object with countries as keys and an array of numbers
+         * as values.
+         */
+        numbers: PropTypes.oneOfType([
+            PropTypes.array,
+            PropTypes.object
```

Comment:
Could you please remind me in what case do we have the numbers without the countries and vice versa?

Reply:
In my personal notes I have that "different environments" have different URLs hardcoded. One of the URLs returns an object while another returns an array of numbers. The env hcv-meetjitsi returns an object but the env hipchat returns an array. I don't remember where those environments are.